docs/content/developers/design-keuzes.md

```diff
@@ -233,9 +233,10 @@ Een voorbeeld response is dan:
 {
     "type": "URI: https://ref.tst.vng.cloud/ref/fouten/ValidationError/",
     "title": "Ongeldige gegevens",
-    "status": 400,
+
+"status": 400,
```

Comment:
indentatie is verdwenen

Reply:
Zie [deze opmerking](https://github.com/VNG-Realisatie/gemma-zaken/pull/411/files/acaf8106f080f15c7580356917f227b4fa3d7b7b#r220491359)